pull/0027_Create_Community_Advisory_Board

```diff
@@ -0,0 +1,51 @@
+- Feature Name: Create a Community Advisory Board
+- Start Date: 2020-09-22
+- SEP Status: Draft
+- SEP PR: (leave this empty)
+- Salt Issue: (leave this empty)
+
+## Summary
+To help give the community a voice for sharing its opinions and driving the direction of Salt, we’re establishing a Community Advisory Board. The board will represent the Salt contributor community, meet regularly with the Salt Open team, and influence changes to Salt and its community.
+
+## Motivation
+The Salt community wants to be more involved in decisions that impact the Salt open project. Currently, this process is ad hoc based on who is in attendance at Open Hours or is vocal in Slack, IRC, or the comments of issues and PRs. Community members often miss change announcements or don’t realize that decisions that impact them are being made. They want a say in Salt Enhancement Proposals (SEPs), Working Groups, technical decisions, and best practices. 
+Creating an Advisory Board will bring structure and process to influencing the Salt project. It will give community members a voice to discuss, agree, and disagree with proposed changes and decisions. Since the board will meet on a regular cadence, board members and those they represent will be able to plan, prepare for, and prioritize proposed changes.
+The Advisory Board’s main goal will be to grow and improve community collaboration and participation. Its members will work with users, contributors, and the Salt Core team. It will not be a replacement for current contributor involvement opportunities, but a supplement to those opportunities. 
+
+## Design 
+
+### Board Logistics
+- The board will be made up of 5 members from diverse technical areas and geographic locations of the Salt contributor community.
+- The board will meet via video call with the SaltStack Core team every month. 
+- Meetings will be moderated by the Community Manager.
+- Meeting agendas will be set and shared prior to meetings.
+- Board meetings will be recorded and require someone to create meeting minutes that will be shared with the greater Salt community.
+
+### Membership
+- Membership will be determined by the combined opinions of the community and the Salt Core team.
+- All active contributors are eligible to join the Advisory Board 
+- Anyone who is interested in joining the board can nominate themselves. They will need to get 3 references from other contributors. The Salt Core team will weigh this feedback when determining who to join the board.
+
+### Expectations
+- All Advisory Board members must adhere to the Contributor Code of Conduct. 
+- Members will discuss and influence the following areas: SEPs, Working Groups, technical decisions, and best practices.
+- The anticipated time commitment for board membership is 3-5 hours per month.
+
+## Alternatives
+- Maintain the community-core team interactions as they are.
+- Create another formal venue for contributors to influence the project.
+
+## Unresolved questions
```

Comment:
Here are some of my thoughts, but not opening a formal review. I would like to hear from others as well who are interested in applying for the board.

- Frequency of meeting and how often is really a question for the board itself, but please give us input here. Would anyone join something as often as monthly? Is quarterly meeting enough?

- Candidates need to be active contributors to the project. I asked in the Open Hour MAR-04 to submit an email to me at sage@saltstack.com with why you want to join, your thoughts on what the purpose of the board is, and some sort of identification (I put resume or LinkedIn profile, but I would also take other forms of identification such as github/gitlab profile).

- Keeping the board's work organized and prioritized can also be determined by the board itself, but we will give a place for this occur. ATM I would expect this to be here in a new directory: https://github.com/saltstack/community. and the sharing of agenda and details of meeting minutes would adhere to our current practice of 2 days prior agenda items will be set and at the end of meetings we will leave ad-hoc time and suggestions of new agenda items, the board can park agenda items for later dates as they fit. Meeting minutes will be posted 2 business days after the meeting and documentation like this will live in the community repo.

- Board member's tenure is likely to be decided by the board itself, as well as if specific positions are assigned or tasked to be held for a tenured amount of time. At this time, this detail can be decided by the board members in collaboration with the Core Team.

- We will continue to use the communication platforms of Slack, IRC, public google groups and the community repo. The board will determine official communication once formed.

- The board will take direction and work in collaboration with the Project's Core team on activities, projects, goals and objectives. A project charter template exists for the Working Groups and likely this would be similarly adopted fo the CAB and created with its members once formed. It can be found here: https://github.com/saltstack/community/blob/master/working_groups/community-temp-proj-chrtr.md

Reply:
Going to use your feedback as a jumping-off point.

> Frequency of meeting and how often is really a question for the board itself, but please give us input here. Would anyone join something as often as monthly? Is quarterly meeting enough?

I think that this would depend on release cadence, but to me quarterly feels too infrequent. Monthly seems reasonable to me.

> Candidates need to be active contributors to the project. I asked in the Open Hour MAR-04 to submit an email to me at sage@saltstack.com with why you want to join, your thoughts on what the purpose of the board is, and some sort of identification (I put resume or LinkedIn profile, but I would also take other forms of identification such as github/gitlab profile).

Done.

> Keeping the board's work organized and prioritized can also be determined by the board itself, but we will give a place for this occur. ATM I would expect this to be here in a new directory: https://github.com/saltstack/community. and the sharing of agenda and details of meeting minutes would adhere to our current practice of 2 days prior agenda items will be set and at the end of meetings we will leave ad-hoc time and suggestions of new agenda items, the board can park agenda items for later dates as they fit. Meeting minutes will be posted 2 business days after the meeting and documentation like this will live in the community repo.

This seems reasonable, the only question I would have is how access to that repo is managed. It would be nice to have the repo be publicly visible, but have comment/PR submission be restricted to Salt Core Team and CAB members. However, I'm not familiar with GitHub's access control options, and whether or not something like this is doable.

> Board member's tenure is likely to be decided by the board itself, as well as if specific positions are assigned or tasked to be held for a tenured amount of time. At this time, this detail can be decided by the board members in collaboration with the Core Team.

This seems fine as well. I think that definition can be worked out at a later date.

> We will continue to use the communication platforms of Slack, IRC, public google groups and the community repo. The board will determine official communication once formed.

Slack does have the ability to create invite only channels, so Slack can be used for this kind of communication.

> The board will take direction and work in collaboration with the Project's Core team on activities, projects, goals and objectives. A project charter template exists for the Working Groups and likely this would be similarly adopted fo the CAB and created with its members once formed. It can be found here: https://github.com/saltstack/community/blob/master/working_groups/community-temp-proj-chrtr.md

:+1: